mlflow/tracking/fluent.py

```diff
@@ -646,7 +646,6 @@ def log_figure(figure, artifact_file):
     .. _plotly.graph_objects.Figure:
         https://plotly.com/python-api-reference/generated/plotly.graph_objects.Figure.html
 
-    :param run_id: String ID of the run.
```

Comment:
Removed this since `log_figure` (and `log_image`) doesn't have a `run_id` argument.